client/app/cloud/project/billing/rights/cloud-project-billing-rights.html

```diff
@@ -91,7 +94,10 @@
                         class="oui-button oui-button_link oui-button_icon-right btn-margin"
                         title="{{:: 'cpb_rights_modify' | translate}}"
                         data-ng-disabled="!BillingRightsCtrl.canChangeContacts()"
-                        data-ng-click="BillingRightsCtrl.openContacts()">
+                        data-ng-click="BillingRightsCtrl.openContacts()"
+                        data-track-on="click"
+                        data-track-name="modification_contact_facturation"
```

Comment:
Maybe you could create a Service with Constant to handle the tracking. This way, it would be easier to update all the tracking if we have to :)

Reply:
Can i add in controller helper ? instead of injecting another service, we can use controller helper ?